daprdocs/content/en/developing-applications/building-blocks/actors/howto-actors.md

```diff
@@ -163,15 +163,17 @@ Refer [api spec]({{< ref "actors_api.md#invoke-reminder" >}}) for more details.
 You can configure the Dapr actor runtime configuration to modify the default runtime behavior.
 
 ### Configuration parameters
+- `entities` - The actor types supported by this host.
```

Comment:
No that is always there and the only actual required field. I'm not sure why we didn't reference it honestly. My guess is because the SDKs abstract it away from us but I felt it was worth calling out given the required reference.